guardian/editorial-production-metrics
# Set up CI script and upgrade node-sass

## What does this change?

This PR brings in the CI behaviour that was previously hardcoded into TeamCity steps into a `scripts/ci.sh` script. It also upgraded the Node version to 10 from the now-deprecated 6.

Once this is merged in we can upgrade the deprecated dependencies more easily. 

## How to test

Clone the repository, checkout the branch, run `nvm use` to use node 10 and run `./scripts/ci.sh`. This should exit cleanly.

## How can we measure success?

The script passes in CI and the app still runs as intended.
